Src/Sarif.PatternMatcher.Cli/StressCommand.cs

```diff
@@ -66,10 +69,11 @@ public void LogToolNotification(Notification notification, ReportingDescriptor _
     internal class StressCommand : CommandBase
     {
         private static int filesScanned = 0;
-        private IEnumerable<string> s_configurationFiles;
-        private readonly List<Tuple<string, long, long>> fileDataTupleList = new List<Tuple<string, long, long>>();
+        private readonly List<Tuple<string, long, long, double, double>> fileDataTupleList = new List<Tuple<string, long, long, double, double>>();
```

Comment:
The tuple has been expanded to accommodate two new fields: CPU utilization and peak memory usage.

Reply:
At this point could we maybe just have a Model class to contain the parameters and if we need to tweak the parameters it won't affect code signatures. It starts getting really awkward with that many tuple fields.

Reply:
@schlaman-ms Good suggestion. Done.